src/pages/Task.tsx

```diff
@@ -71,13 +90,23 @@ export const Task: React.FC = () => {
     status === TaskStatus.Succeeded ||
     status === TaskStatus.SetupFailed ||
     status === TaskStatus.SystemFailed ||
-    status === TaskStatus.TestTimedOut
+    status === TaskStatus.TestTimedOut ||
+    error
   ) {
     stopPolling();
   }
```

Comment:
per the discussion on prev pr with john, we decided to stop polling altogether and let the page infinitely poll until the component unmounts

Reply:
sounds good. I can remove the status conditions but I think the error condition to stop polling makes sense. If one polled request fails, it's likely that the next polled request will also fail and produce the same error banner.